diff --git a/dist/esm/constants/platform.js b/dist/esm/constants/platform.js
index d38e877e8f7c318faa8c0236d219260c6367cf9c..442c4535255690b9792e259bec7468c23948e8cb 100644
--- a/dist/esm/constants/platform.js
+++ b/dist/esm/constants/platform.js
@@ -7,7 +7,7 @@ const isWeb = process.env.TAMAGUI_TARGET === "web";
 const isWindowDefined = typeof window !== "undefined";
 const isSSR = isWeb && !isWindowDefined;
 const isClient = isWeb && isWindowDefined;
-const isRSC = process.env.ENABLE_RSC ? import.meta.env ? import.meta.env.SSR : false : false;
+const isRSC = false;
 const idFn = () => {
 };
 const useIsomorphicLayoutEffect = isRSC ? idFn : isSSR ? useEffect : useLayoutEffect;
